### PR TITLE
Add support for object type spread

### PR DESCRIPTION
**Pull request by conartist6**
| Q                        | A
| ------------------------ | ---
| Patch: Bug Fix?          | N
| Major: Breaking Change?  | N
| Minor: New Feature?      | Y
| Deprecations?            | N
| Spec Compliancy?         | Y (No official spec; matches flow behavior)
| Tests Added/Pass?        | Y
| Fixed Tickets            | 
| License                  | MIT
| Doc PR                   | N
| Dependency Changes       | https://github.com/babel/babylon/pull/418

### CODE REVIEW

File: packages/babel-generator/src/generators/flow.js [hunk 1]
```diff
@@ -331,6 +331,12 @@ export function ObjectTypeProperty(node: Object) {
   this.print(node.value, node);
 }
 
+export function ObjectTypeSpreadProperty(node: Object) {
+  this._variance(node);
```
Comment (samwgoldman) [inline packages/babel-generator/src/generators/flow.js]: Since this node type can't have a variance annotation, this line can be removed.

**Comment by samwgoldman** [comment]
Thanks! Will you also add a test to babel-plugin-transform-flow-strip-types? I don't think any code changes are necessary, but it'd be nice to have the test.

File: packages/babel-generator/src/generators/flow.js (hunk 1 above)
Comment (conartist6) [inline packages/babel-generator/src/generators/flow.js]: Yep

**Comment by samwgoldman** [comment]
Looks good to me! WDYT @danez @hzoo?

**Comment by voideanvalue** [comment]
Any chance this could this be backported to 6.x, similar to babel/babylon#418?

**Comment by existentialism** [comment]
@voideanvalue this was done in https://github.com/babel/babel/pull/5653, we'll release it soon!

**Comment by voideanvalue** [comment]
@existentialism Sweet. Any ETA on a new release?

**Pull request merged**